src/scripts/services/dimBungieService.factory.js

```diff
@@ -346,72 +242,29 @@ function BungieService($rootScope, $q, $timeout, $http, $state, dimState, toaste
     }
   }
 
-
-  /************************************************************************************************************************************/
-
   function getStores(platform) {
-    var data = {
-      token: null,
-      membershipId: null
-    };
-
-    var addTokenToData = assignResultAndForward.bind(null, data, 'token');
-    var addMembershipIdToData = assignResultAndForward.bind(null, data, 'membershipId');
-    var addCharactersToData = assignResultAndForward.bind(null, data, 'characters');
-    var getMembershipPB = getMembership.bind(null, platform);
-    var getCharactersPB = getCharacters.bind(null, platform);
-
-    var promise = getBungleToken()
-      .then(addTokenToData)
-      .then(getMembershipPB)
-      .then(addMembershipIdToData)
-      .then(getCharactersPB)
-      .then(addCharactersToData)
-      .then(function() {
-        var promises = [
-          getDestinyInventories(data.token, platform, data.membershipId, data.characters),
-          getDestinyProgression(data.token, platform, data.membershipId, data.characters)
+    return $q.all([
```

Comment:
nice... firing both async. should help slightly with load?

nice work on refactoring these functions btw!